### PR TITLE
Fixed issues when user was seeing full stace trace in prod

### PR DESCRIPTION
**Pull request by fareedmohammed**

### CODE REVIEW

**Comment by astroash** [comment]
Ran this locally with `NODE_ENV=production` and stack trace is still rendered on error page:
Screenshot 2020-07-24 at 16 33 16

File: server/errorHandler.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const logger = require('../log.js')
+
+const production = process.env.NODE_ENV === 'production'
```
Comment (astroash) [inline server/errorHandler.js:3]: Is there a reason we're only removing the stack trace from the UI in `production`? I think for both  development and prod purposes we can log the error in to the server, and remove the additional code of checking the enviroment in this handler

Comment (fareedmohammed) [reply server/errorHandler.js:3]: They do that in other projects such as Pathfinder and Use Of Force, so I just followed the same pattern here.

File: server/views/pages/error.ejs [hunk 2]
```diff
@@ -14,11 +14,11 @@
 
         <p class="govuk-caption-l govuk-!-margin-bottom-6"><%= error.status %></p>
 
-        <p class="govuk-body"><%= error.stack %></p>
+        <p class="govuk-body"><%= stack %></p>
```
Comment (nickmcmahon01) [inline server/views/pages/error.ejs:17]: Why would we print anything here at all?

File: server/views/pages/error.ejs [hunk 3]
```diff
@@ -14,11 +14,11 @@
 
         <p class="govuk-caption-l govuk-!-margin-bottom-6"><%= error.status %></p>
```
Comment (nickmcmahon01) [inline server/views/pages/error.ejs:15]: What is error status? we shouldn't be leaking any technical information.

File: server/errorHandler.js (hunk 1 above)
Comment (nickmcmahon01) [reply server/errorHandler.js:3]: all of our code should behave the same across all environments and locally. I'll raise a ticket to remove the rest of these switches, but these ones can be tackled in this PR.

**Pull request merged**